code/modules/paperwork/paperbin.dm

```diff
@@ -91,7 +91,7 @@
 	else
 		to_chat(user, "<span class='warning'>[src] is empty!</span>")
 	add_fingerprint(user)
-	return ..()
+	// return ..() // Breaks the paperbin where the bin drops to the ground from your hand, bag, etc, when taking a paper or pen

```

Comment:
i think the standard is to just remove this line, we dont need commented code

Reply:
It needs a memorial.

Reply:
This is not the standard.

Reply:
leaving a comment is fine, leaving commented-out code is not usually a good idea

Reply:
What @SpaceManiac said

Reply:
remove the code it doesn't matter much but add a comment up above that this attack hand doesn't respect return value of parent.